docs/subnets/reference-cli-commands.md

```diff
@@ -0,0 +1,517 @@
+    --mainnet mainnet             join on mainnet
+    --nodeID string               set the NodeID of the validator to check
+    --plugin-dir string           file path of avalanchego's plugin directory
+    --print                       if true, print the manual config without prompting
+    --skip-whitelist-check        if true, skip the whitelist check
+    --testnet testnet             join on testnet (alias for `fuji`)
+```
+
+### list
+
+The subnet list command prints the names of all created subnet
+configurations.
+
+**Usage:**
+
+`avalanche subnet list [flags]`
+
+**Flags:**
+
+```
+-h, --help   help for list
+```
+
+### publish
+
+Publish the subnet's VM to a repository
+
+**Usage:**
+
+`avalanche subnet publish [subnetName] [flags]`
+
+**Flags:**
+
+```
+    --alias string              We publish to a remote repo, but identify the repo locally under a user-provided alias (e.g. myrepo).
+    --force                     If true, ignores if the subnet has been published in the past, and attempts a forced publish.
+-h, --help                      help for publish
+    --no-repo-path string       Do not let the tool manage file publishing, but have it only generate the files and put them in the location given by this flag.
+    --repo-url string           The URL of the repo where we are publishing
+    --subnet-file-path string   Path to the Subnet description file. If not given, a prompting sequence will be initiated.
+    --vm-file-path string       Path to the VM description file. If not given, a prompting sequence will be initiated.
+```
+
+### stats
+
+Show validator statistics for the given subnet
+
+**Usage:**
+
+`avalanche subnet stats [subnetName] [flags]`
+
+**Flags:**
+
+```
+    --fuji fuji         print stats on fuji (alias for `testnet`)
+-h, --help              help for stats
+    --mainnet mainnet   print stats on mainnet
+    --testnet testnet   print stats on testnet (alias for `fuji`)
+```
+
+## Network
+
+The network command suite provides a collection of tools for managing
+local subnet deployments.
+
+When a subnet is deployed locally, it runs on a local, multi-node
+Avalanche network. Deploying a subnet locally will start this network
+in the background. This command suite allows you to shutdown,
+restart, and clear that network.
+
+This network currently supports multiple, concurrently deployed
+subnets.
+
+### clean
+
+The network clean command shuts down your local, multi-node network. All
+the deployed subnets will shutdown and delete their state. The network
+may be started again by deploying a new subnet configuration.
+
+**Usage:**
+
+`avalanche network clean [flags]`
+
+**Flags:**
+
+```
+    --hard   Also clean downloaded avalanchego and plugin binaries
+-h, --help   help for clean
+```
+
+### start
+
+The network start command starts a local, multi-node Avalanche network
+on your machine.
+
+By default, the command loads the default snapshot. If --snapshot-name flag
```

Comment:
I think if we start mentioning snapshots here, either we'd maybe want to have some explanation what this does/is, or link to somewhere.

Reply:
https://github.com/ava-labs/avalanche-cli/issues/483